### PR TITLE
Create description of managing notifications

### PR DESCRIPTION
**Pull request by r5atom**
## Overview
**TL;DR**
* This file can be inserted somewhere to return instructions to _manage notifications_.

Instructions to silence notifications became redundant after the access method changed.  
Closes #468 

### Questions
* What is a good location to insert these instructions?

### Next Steps
* Include it in one of the course _Tell Me Why_ sections

### Review
Suggestions welcome @beardofedu 

---
newbie alert: @r5atom's first pull request with his account age being <24h

### CODE REVIEW

File: _includes/tell-me-why/manage-notifications.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+##Managing Notifications
+Once you've commented on an _issue_ or _pull request_, email notifications will start to pour in. By default notifications will be send by email when there is activity in the thread.
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: "By default, notifications will be sent"

File: _includes/tell-me-why/manage-notifications.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+##Managing Notifications
+Once you've commented on an _issue_ or _pull request_, email notifications will start to pour in. By default notifications will be send by email when there is activity in the thread.  
+You can also choose to receive notifications of a full repository. Whenever there is a new _issue_, _pull request_ or _comment_ posted you will get a notification.
+
+###You can silence or unmute notifications of _individual threads_.
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: I would change "individual threads" to "individual issues and pull requests"

File: _includes/tell-me-why/manage-notifications.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Go to the thread.
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: "Go to the thread." -> "Go to the issue or pull request."

File: _includes/tell-me-why/manage-notifications.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+2. Click the _(Un)subscribe_ button on the right, under _Notifications_  
+Underneath the button a short description explains the current notification status.
+
+###You can _customize the notifications_ in your settings.
+
+1. Go to your profile icon
+2. Go to _settings_
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: "settings" should be "Settings" to mimic the menu option

File: _includes/tell-me-why/manage-notifications.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+Once you've commented on an _issue_ or _pull request_, email notifications will start to pour in. By default notifications will be send by email when there is activity in the thread.  
+You can also choose to receive notifications of a full repository. Whenever there is a new _issue_, _pull request_ or _comment_ posted you will get a notification.
+
+###You can silence or unmute notifications of _individual threads_.
+
+1. Go to the thread.
+2. Click the _(Un)subscribe_ button on the right, under _Notifications_  
+Underneath the button a short description explains the current notification status.
+
+###You can _customize the notifications_ in your settings.
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: "You can _customize_ notifications in Settings"

File: _includes/tell-me-why/manage-notifications.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Go to your profile icon
+2. Go to _settings_
+3. Select _Notification_ from the menu on the left.
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: "_Notifications_" to mimic the displayed page

File: _includes/tell-me-why/manage-notifications.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+1. Go to the thread.
+2. Click the _(Un)subscribe_ button on the right, under _Notifications_  
+Underneath the button a short description explains the current notification status.
+
+###You can _customize the notifications_ in your settings.
+
+1. Go to your profile icon
+2. Go to _settings_
+3. Select _Notification_ from the menu on the left.  
+Here you can adjust your notification preferences
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: We can add a link to the word preferences to this GitHub Help article: https://help.github.com/articles/managing-notification-delivery-methods/

File: _includes/tell-me-why/manage-notifications.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+###You can choose to receive notification from a _repository_.
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: Change "You can choose to receive notifications from a _repository_"

File: _includes/tell-me-why/manage-notifications.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+* _**Watch**_ : You will receive a notification when a **new** _issue_, _pull request_ or _comment_ is posted, and when an _issue_ is **closed** or a _pull request_ is **merged**.
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: "You will receive notifications..."

File: _includes/tell-me-why/manage-notifications.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+3. Select _Notification_ from the menu on the left.  
+Here you can adjust your notification preferences
+
+###You can choose to receive notification from a _repository_.
+
+* _**Watch**_ : You will receive a notification when a **new** _issue_, _pull request_ or _comment_ is posted, and when an _issue_ is **closed** or a _pull request_ is **merged**.  
+* _**Not watching**_ : Stop receiving notifications, but @ mentions will still alert you.  
+* _**Ingore**_ : Stops all notifications.  
+
+###You can _review the notifications_ for repositories you are watching
```
Comment (beardofedu) [inline _includes/tell-me-why/manage-notifications.md]: _review notifications_

File: _includes/tell-me-why/manage-notifications.md (hunk 2 above)
Comment (r5atom) [inline _includes/tell-me-why/manage-notifications.md]: 👍  I wasn't sure about this. I initially settled for short, but see that explicitly mentioning _"issue"_ and _"pull request"_ might be less confusing.

**Comment by r5atom** [comment]
I will now look for a page where this can live.

**Comment by r5atom** [comment]
@beardofedu I think this _notification_ section is best suited [early on](https://services.github.com/on-demand/intro-to-github/02/) in the [101 course](https://services.github.com/on-demand/intro-to-github/). But [this page](https://github.com/github/training-kit/blob/master/paths/github-cli/02-local-configurations.md) already contains a _tell-me-why_ section.  

How is this section incorporated?   
Can _"includes"_ live side-by-side with already existing markup?

**Comment by hectorsector** [comment]
> How is this section incorporated?

A _tell-my-why_ can be incorporated either inline or as an `include`.  [Here's an inline example](https://github.com/github/training-kit/blame/master/paths/intro-to-github/02-join-class-repo.md#L15) and [here's one with `include`](https://github.com/github/training-kit/blame/master/paths/github-desktop/05-cloning-repository-locally.md#L16). Since the page you identified already contains a _tell-me-why_, you can either (a) add your content inline to the already existing one, or (b) move the existing content to its own include and then add your new content as a second include. I'm happy to help ✋ if you'd like me to.

> I think this notification section is best suited early on in the 101 course

Good eye, @r5atom! I agree the place you identified is a good fit, since when a user gets added as a collaborator, they'll begin receiving notifications. So it's a good time to tell them how to control those notifications.

**Comment by r5atom** [comment]
> you can either 
(a) add your content inline to the already existing one, or 
(b) move the existing content to its own include

I opted for **(b)**. 
_Creating issues_ and _managing notifications_ seem separate topics and two files maintain a modular design.

Let me know what you think. I think it is close to 🏁 , no?

File: _includes/tell-me-why/manage-notifications.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
+##Managing Notifications
+Once you've commented on an _issue_ or _pull request_, email notifications will start to pour in. By default, notifications will be send by email when there is activity in the thread.
```
Comment (hectorsector) [inline _includes/tell-me-why/manage-notifications.md]: > notifications will be send by email

Should read:

> notifications will be sent by email

File: _includes/tell-me-why/manage-notifications.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+1. Go to your profile icon
+2. Go to _Settings_
+3. Select _Notifications_ from the menu on the left.  
+Here you can adjust your notification [preferences](https://help.github.com/articles/managing-notification-delivery-methods/). 
+
+###You can choose to receive notifications from a _repository_.
+
+* _**Watch**_ : You will receive a notifications when a **new** _issue_, _pull request_ or _comment_ is posted, and when an _issue_ is **closed** or a _pull request_ is **merged**.  
+* _**Not watching**_ : Stop receiving notifications, but @ mentions will still alert you.  
+* _**Ingore**_ : Stops all notifications.
```
Comment (hectorsector) [inline _includes/tell-me-why/manage-notifications.md]: > Ingore

Should read:

> Ignore

File: _includes/tell-me-why/manage-notifications.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+###You can _review notifications_ for the repositories you are already watching.
+
+1. Go to your profile icon
+2. Go to _settings_
+3. Select _Notification_ from the menu on the left.
+4. Click on the [repositories you’re watching](https://github.com/watching) link.
+5. Select the _Watching_ tab.
+6. Click the _(Un)watch_ button to disable or enable notifications for that repository.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ocQldxF7fM" frameborder="0" allowfullscreen></iframe>
```
Comment (hectorsector) [inline _includes/tell-me-why/manage-notifications.md]: This video doesn't work for me 

screen shot 2017-02-21 at 3 28 14 pm

**Comment by r5atom** [comment]
I've fixed some grammar/typos. Well spotted @hectorsector !

---
_[off topic]_
> This video doesn't work for me

I probably would have caught the video issue earlier if I had seen the generated web page. 
Is there a way to preview the generated website instead of the markup?

**Comment by beardofedu** [comment]
@r5atom here are the instructions for setting up a local instance of the website. 

https://github.com/github/training-kit#building-and-packaging

Thanks again for all of the help with this project. 👍

**Comment by r5atom** [comment]
@beardofedu I should have read the manual. :man_facepalming: 

Unfortunately I don't have the appropriate privileges on my machine to run these scripts. I'll leave it for now and @hectorsector should go ahead 👍  merging it.

Thanks to all for your help and encouragement. This is maybe the best way of learning the basics!

**Comment by hectorsector** [comment]
Thanks @r5atom. Your ✨ notifications are [now live](https://services.github.com/on-demand/intro-to-github/02/)!

Thanks for the great feedback, we ❤️ your contribution and hope to see some more.

**Pull request merged**